tests/test_coronagraphy.py

```diff
@@ -175,3 +175,33 @@ def test_lyot_coronagraph():
 	# Checks performance of the coronagraph and if the focal length does not introduce artifacts
 	assert (wf_foc.power.max() / norm) < 5e-3
 	np.testing.assert_allclose(wf_foc.power, wf_foc2.power)
+
+
+def test_knife_edge_coronagraph():
+	grid = make_pupil_grid(64, 1.1)
+	aperture = circular_aperture(1)(grid)
+
+	focal_grid = make_focal_grid(q=5, num_airy=3)
+
+	prop = FraunhoferPropagator(grid, focal_grid)
+	lyot_stop = circular_aperture(0.95)(grid)
+
+	wf = Wavefront(aperture)
+	wf.total_power = 1.0
+	norm = prop(wf).power.max()
+
+	directions = ['+x', '-x', '+y', '-y']
+	for direction in directions:
+		knife_left = KnifeEdgeLyotCoronagraph(grid, direction=direction, apodizer=None, lyot_stop=lyot_stop)
```

Comment:
`knife_left`? Why the "left"?

Reply:
This was a remnant of older code. I changed it now.